src/AutoRest.CSharp/Common/AutoRest/Plugins/NewProjectScaffolding.cs

```diff
@@ -467,12 +332,20 @@ private string GetSln()
 		{{8E9A77AC-792A-4432-8320-ACFD46730401}}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{{8E9A77AC-792A-4432-8320-ACFD46730401}}.Release|Any CPU.Build.0 = Release|Any CPU
 ";
-            if (_isAzureSdk)
+            if (_isAzureSdk && !Configuration.AzureArm)
             {
                 slnContent += @"		{{ECC730C1-4AEA-420C-916A-66B19B79E4DC}}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{{ECC730C1-4AEA-420C-916A-66B19B79E4DC}}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{{ECC730C1-4AEA-420C-916A-66B19B79E4DC}}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{{ECC730C1-4AEA-420C-916A-66B19B79E4DC}}.Release|Any CPU.Build.0 = Release|Any CPU
+";
+            }
+            if (Configuration.AzureArm && Configuration.Namespace.StartsWith("Azure.ResourceManager"))
```

Comment:
In autorest.csharp, there are no proper project or package reference. If we generate sample.csproj and added in solution file, then the pipeline would have many syntax error. I restrict the namespace to "Azure.ResourceManager",  so it would only generate sample's csproj file and added to the solution file under real sdk management generation environment.

Reply:
How about adding a configuration "skip-test-project-generation" to skip the test projects generation? Similar to https://github.com/Azure/autorest.csharp/blob/c27e20febd3ddd94b80da58571772cf9b1a3c5f3/src/AutoRest.CSharp/Common/Input/Configuration.cs#L32
That will be clearer.

Reply:
used the mocked way  instead of using config to avoid generation of somefile.